src/helpers/AbstractWorkspace.ts

```diff
@@ -0,0 +1,138 @@
+// Copyright (c) 2022. Consensys Software Inc. All rights reserved.
+// Licensed under the MIT license.
+
+import {Constants} from '@/Constants';
+import {showQuickPick} from '@/helpers/userInteraction';
+import {Telemetry} from '@/TelemetryClient';
+import glob from 'glob';
+import * as path from 'path';
+import {Uri, workspace} from 'vscode';
+
+/**
+ * The [glob](https://github.com/isaacs/node-glob#glob-primer) pattern to match Truffle/Other config file names.
+ */
+export const TRUFFLE_CONFIG_GLOB = 'truffle-config{,.*}.js';
+export const HARDHAT_CONFIG_GLOB = 'hardhat.config{,.*}.ts';
```

Comment:
We would also need to include Hardhat `js` config files, _i.e._, we can use the following glob `hardhat.config{,.*}.{js,ts}`.

Reply:
ok I'll make that change I just think the majority of HH projects are TS based. I don't think they even give you a choice when bootstrapping a project to use JS anymore?

Reply:
As of HardHat `v2.12.2` they do

<img width="594" alt="image" src="https://user-images.githubusercontent.com/4592980/202042507-6afa7b8d-5475-4c51-b023-88864795b427.png">

Reply:
I will change the glob. thanks! 👍🏻